VotingApplication/VotingApplication.Web/Scripts/Controllers/CreateBasicPageController.js

```diff
@@ -1,13 +1,21 @@
 ﻿(function () {
-        angular.module('GVA.Creation').controller('CreateBasicPageController', ['$scope', 'AccountService', function ($scope, AccountService) {
+    angular
+        .module('GVA.Creation')
+        .controller('CreateBasicPageController', ['$scope', 'AccountService', 'PollService', function ($scope, AccountService, PollService) {
 
-        $scope.openLoginDialog = function () {
-            AccountService.openLoginDialog($scope);
-        }
+            $scope.openLoginDialog = function () {
+                AccountService.openLoginDialog($scope);
+            };
 
-        $scope.openRegisterDialog = function () {
-            AccountService.openRegisterDialog($scope);
-        }
+            $scope.openRegisterDialog = function () {
+                AccountService.openRegisterDialog($scope);
+            };
 
-    }]);
+            $scope.createPoll = function (question) {
+                PollService.createPoll(question, function (data) {
+                    window.location.href = "/Manage/" + data.ManageId;
```

Comment:
Just tested this after the merge and it's not working. Will fix and add to this PR.